### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by Salimer**
## What does this PR do?

This PR is raised to implement the following features 👍:

- [x] Set up React Testing Library.
- [x] Write unit tests for the files operate.js and calculate.js using Jest.
- [x] Create unit tests for all React components:
- [x] Run your full tests suite using Jest. All tests should pass.

## Description of the completed tasks 🙌

- [x] Added the HTML for the structure. 
- [x] Added the SCSS file for the styling.
- [x] Added the js components for React.
- [x] Used the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 💯

### CODE REVIEW

File: src/__tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import calculate from '../logic/calculate';
```
Comment (FrederickMih) [inline src/__tests__/calculate.test.js:1]: - [ ] Hi @Salimer, it seems we are missing the scenario when the button name is "**=**" and "**0**". Let's kindly address the issue and submit our code once more.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: :raising_hand: @Salimer, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- PR is professional :+1: 
- All written test cases are running without errors :+1: 
- Linter checks passed :heavy_check_mark: 


## Required Changes ♻️
**N/A**
_Check the comments under the review._

## Optional suggestions
**N/A**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Salimer** [comment]
Thank you @FrederickMih for the thorough review. We addressed the following: 
- Add a scenario when the button name is "="
- Add a scenario when the button name is "0"

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Salimer and @Mar12358 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**